docs/docs/guide/08-loan/00-intro.md

```diff
@@ -0,0 +1,86 @@
+build will be powered by a blockchain, which provides a decentralized and
+immutable record of all transactions. This ensures that the platform is
+transparent, secure, and resistant to fraud.
+
+A loan is a financial transaction in which one party, the borrower, receives a
+certain amount of assets, such as money or digital tokens, and agrees to pay
+back the loan amount plus a fee to the lender by a predetermined deadline. To
+secure the loan, the borrower provides collateral, which may be seized by the
+lender if the borrower fails to pay back the loan as agreed.
+
+A loan has several properties that define its terms and conditions.
+
+The `id` is a unique identifier that is used to identify the loan on a
+blockchain.
+
+The `amount` is the amount of assets that are being lent to the borrower.
+
+The `fee` is the cost that the borrower must pay to the lender for the loan.
+
+The `collateral` is the asset or assets that the borrower provides to the lender
+as security for the loan.
+
+The `deadline` is the date by which the borrower must pay back the loan. If the
+borrower fails to pay back the loan by the deadline, the lender may choose to
+liquidate the loan and seize the collateral.
+
+The `state` of a loan describes the current status of the loan and can take on
+several values, such as `requested`, `approved`, `paid`, `cancelled`, or
+`liquidated`. A loan is in the `requested` state when the borrower first submits
+a request for the loan. If the lender approves the request, the loan moves to
+the `approved` state. When the borrower repays the loan, the loan moves to the
+`paid` state. If the borrower cancels the loan or the lender denies the request,
+the loan moves to the `cancelled` state. If the borrower is unable to pay back
+the loan by the deadline, the lender may choose to liquidate the loan and seize
+the collateral. In this case, the loan moves to the `liquidated` state.
+
+In a loan transaction, there are two parties involved: the borrower and the
+lender. The borrower is the party that requests the loan and agrees to pay back
+the loan amount plus a fee to the lender by a predetermined deadline. The lender
+is the party that approves the loan request and provides the borrower with the
+loan amount.
+
+As a borrower, you should be able to perform several actions on the loan platform.
+These actions may include:
+
+* requesting a loan,
+* canceling a loan,
+* repaying a loan.
+
+Requesting a loan allows you to specify the terms and conditions of the loan,
+including the amount, the fee, the collateral, and the deadline for repayment.
+If you cancel a loan, you can withdraw your request for the loan before it is
+approved or funded. Repaying a loan allows you to pay back the loan amount plus
+the fee to the lender in accordance with the loan terms.
+
+As a lender, you should be able to perform two actions on the platform:
+
+* approving a loan
+* liquidating a loan.
```

Comment:
What about denying the loan request ? This was mentioned earlier in : 


> If the borrower cancels the loan or the lender denies the request, 
the loan moves to the `cancelled` state.

Reply:
> If the borrower cancels the loan before it is, the loan moves to the `cancelled` state.

Isn't there a word missing after `it is` ?

Reply:
Yeah, sorry about that 🫤 fixed.